source/catalog.html

```diff
@@ -0,0 +1,362 @@
+        </h2>
+        <div class="filter__wrapper">
+          <form class="filter__form">
+            <label for="form-file"
+              ><img
+                src="img/catalog/photo-7.jpg"
+                alt="Фото зимней дороги"
+                class="filter-image"
+                width="280"
+                height="374"
+              />:</label
+            >
+            <input
+              class="visually-hidden filter__form-file"
+              type="file"
+              id="form-file"
+              name="form-file"
+            />
+            <div class="filter__tools">
+              <button>
+                <img
+                  src="img/catalog/crop.svg"
+                  alt="Иконка: вырезать"
+                  class="filter__tools-image"
+                  width="44"
+                  height="44"
+                />
+              </button>
+
+              <!-- Range -->
+              <fieldset class="filter-item">
+                <div class="range-filter">
+                  <div class="scale">
+                    <div class="bar" style="width: 0%; margin-left: 0%"></div>
+                  </div>
+                  <div class="toggle" tabindex="0" style="left: 0%"></div>
+                </div>
+              </fieldset>
+
+              <button>
+                <img
+                  src="img/catalog/fill.svg"
+                  alt="Иконка: заливка"
+                  class="filter__tools-image"
+                  width="48"
+                  height="43"
+                />
+              </button>
+
+              <!-- Range -->
+              <fieldset class="visually-hidden filter-item">
+                <div class="range-filter">
+                  <div class="scale">
+                    <div class="bar" style="width: 0%; margin-left: 0%"></div>
+                  </div>
+                  <div class="toggle" tabindex="0" style="left: 0%"></div>
+                </div>
+              </fieldset>
+
+              <button>
+                <img
+                  src="img/catalog/contrast.svg"
+                  alt="Иконка: контраст"
+                  class="filter__tools-image"
+                  width="32"
+                  height="32"
+                />
+              </button>
+
+              <!-- Range -->
+              <fieldset class="visually-hidden filter-item">
+                <div class="range-filter">
+                  <div class="scale">
+                    <div class="bar" style="width: 0%; margin-left: 0%"></div>
+                  </div>
+                  <div class="toggle" tabindex="0" style="left: 0%"></div>
+                </div>
+              </fieldset>
+            </div>
+
+            <div class="filter__button">
+              <button class="button filter__button-post" type="submit">
+                Запостить
+              </button>
+              <button class="button filter__button-cancel" type="button">
```

Comment:
```suggestion
              <button class="button filter__button-cancel" type="reset">
```
Так, как это кнопка сброса параметров.